### PR TITLE
db410c: Debian SD Boot Instructions

### PR DESCRIPTION
**Pull request by ric96**
Added instructions to boot Debian from sdcard.

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

Tagging for review:
@ndechesne

### CODE REVIEW

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md [hunk 1]
```diff
@@ -81,3 +82,40 @@ Where:
...
+
+You can now write the boot and rootfs image into the SD card:
+
+```shell
+$ sudo dd if=boot-sdcard-XXX.img of=/dev/XXX7
+$ sudo simg2img linaro-stretch-alip-XXX.img /dev/XXX9
+```
+NOTE: you may need to install the ```simg2img``` package using:
+```shell
+$ sudo apt install simg2img
```
Comment (ndechesne) [inline ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: nope. it's in android-tools-fsutils

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md [hunk 2]
```diff
@@ -81,3 +82,40 @@ Where:
 * you can check the mapping between partition IDs and partition names using `gdisk -l` command
 
 If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
+
+# 4) Booting Debian image
+
+Download boot and root images from:
+  - **Release:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+  - **Snapshot:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
```
Comment (ndechesne) [inline ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: wrong link here

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md (hunk 1 above)
Comment (ric96) [reply ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: seems to be simg2img on debian, ubuntu(zesty artful bionic)
i'll add that too

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md (hunk 2 above)
Comment (ric96) [reply ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: that seems correct, I'll mention the specific file names they have to download

Comment (ric96) [reply ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: oh ok, got it.

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md [hunk 3]
```diff
@@ -81,3 +82,49 @@ Where:
...
+# OR
+$ sudo apt install android-tools-fsutils
+```
+
+Where:
+* /dev/XXX7 represents the 'boot' partition on the SD card
+* /dev/XXX9 represents the 'rootfs' partition on the SD card
+* you can check the mapping between partition IDs and partition names using `gdisk -l` command
+
+If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
```
Comment (Mani-Sadhasivam) [inline ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: Could be

Now insert the SD card into Dragonboard410c and set switch S6-2 (on back side) to ON. Then power ON the Dragonboard410c to see Debian booting from SD card.

Comment (ric96) [reply ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: @Mani-Sadhasivam i'll modify the previous wget in a separate PR, and the issue with wget not accepting wildcards in http seems to be a recent one, afaik used to work before.

I'll modify the last line

**Comment by ndechesne** [comment]
If wildcards don’t work it is a serious issue! Do not remove them here and instead open a bug on ITS.

**Comment by ndechesne** [comment]
hmm. it looks like wildcards work fine... 

e.g. http://builds.96boards.org/snapshots/dragonboard410c/linaro/debian/301/boot-linaro-*-dragonboard-410c-*.img.gz

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md [hunk 4]
```diff
@@ -81,3 +82,49 @@ Where:
...
+
+# 4) Booting Debian image
+
+Download boot and root images from:
+  - **Release:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+    - Boot File: boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-XXX.img.gz
+    - Root File: 	linaro-stretch-alip-qcom-snapdragon-arm64-XXX.img.gz
+
+
+  - **Snapshot:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
```
Comment (ndechesne) [inline ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: the link here still points to 'releases' not 'snapshots'

File: ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md [hunk 5]
```diff
@@ -81,3 +82,49 @@ Where:
...
+    - Boot File: boot-sdcard-linaro-buster-dragonboard-410c-XXX.img.gz
+    - Root File: linaro-buster-alip-dragonboard-410c-XXX.img.gz
+
+
+  For Example:
+  ```shell
+  $ wget http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-20171016-283.img.gz
```
Comment (ndechesne) [inline ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: please use wildcards

Comment (ric96) [reply ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md]: hmmm... seems to be working. looks like the link in the first section are broken and I was thrown away by: " Warning: wildcards not supported in HTTP."
I'll correct the lins

**Pull request merged**